### DISCUSSION TITLE
concourse start up problem

### DISCUSSION THREAD

**Discussion by vasgupta** [concourse/concourse#8656] [General]
Hello, I can't start the concourse web server container. I get this error below, it has something to do with the iptables library. The code for the iptables startup should be `iptables-**legacy** -A INPUT -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT` , but I don't know how to download the older versions of concourse in the container or enter this code in the source code for this container. 

`{"timestamp":"2023-01-03T07:50:42.324113171Z","level":"error","source":"quickstart","message":"quickstart.worker-runner.logging-runner-exited","data":{"error":"Exit trace for group:\ngarden exited with error: Exit trace for group:\ncontainerd-garden-backend exited with error: setup host network failed: appending accept rule for RELATED \u0026 ESTABLISHED connections failed: running [/usr/sbin/iptables -t filter -A CONCOURSE-OPERATOR -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT --wait]: exit status 2: iptables v1.8.7 (nf_tables): Couldn't load match `conntrack':No such file or directory\n\nTry `iptables -h' or 'iptables --help' for more information.\n\ncontainerd exited with nil\n\ndebug exited with nil\nvolume-sweeper exited with nil\ncontainer-sweeper exited with nil\nbaggageclaim exited with nil\nhealthcheck exited with nil\nbeacon exited with nil\n","session":"2"}}`

**Comment by jhderojasUVa** [comment]
It's a WSL2 problem related here https://github.com/microsoft/WSL/issues/7948#issuecomment-1207210311

**Comment by vasgupta** [comment]
Oh hey I just figured it out. Just use version 7.7.1 instead of the version it downloads automatically. To do that, all you gotta do is change the yaml file you download in the first step to include version 7.7.1 . This is what it should look like now. `image: concourse/concourse:7.7.1` 

hope this helps someone who is stuck on it as well.
